src/garage/tf/policies/continuous_mlp_policy.py

```diff
@@ -122,9 +123,9 @@ def get_action(self, observation):
             dict: Empty dict since this policy does not model a distribution.
 
         """
-        action = self._f_prob([observation])
-        action = self.action_space.unflatten(action)
-        return action, dict()
+        actions, agent_infos = self.get_actions(np.array([observation]))
```

Comment:
i thought you already guaranteed the observation is an np.ndarray?

Reply:
I will need to update the documentation to support the different observation types of akro.